app/javascript/src/utilities/test-utils.js

```diff
@@ -8,6 +8,10 @@
 
 import type { ReactWrapper } from 'enzyme'
 
+function openSelect <T> (wrapper: ReactWrapper<T>) {
```

Comment:
Maybe add another one `function selectOption (wrapper, name)` that combines open + click? See if you can reduce some boilerplate with it. `name` would be the text of the option.